lib/trackerBlocking.js

```diff
@@ -395,6 +399,23 @@ function normalizeTrackerRule (trackerRule) {
     return trackerRule
 }
 
+/**
+ * Finds the closest matching tracker entry for the given domain. Returns the
+ * tracking domain if one is found, null otherwise.
+ * @param {Object.<string, TrackerObj>} trackerEntries
```

Comment:
Can this be
```suggestion
 * @param {Record<string, TrackerObj>} trackerEntries
```
Instead?

This should prevent all the weird edge cases of objects so long as we're not relying on them.

Reply:
Cool OK, Done.